### PR TITLE
[MSYS-686] Prep knife-ec2 release v0.16.0

### PR DESCRIPTION
**Pull request by piyushawasthi**
Signed-off-by: piyushawasthi <piyush.awasthi@msystechnologies.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ Note: this log contains only changes from knife-ec2 release 0.8.0 and later
 prior to release 0.8.0, please visit the [source repository](https://github.com/chef/knife-ec2/commits).
 
 
+## Latest Release 0.16.0 (2017-10-23)
```
Comment (NimishaS) [inline CHANGELOG.md]: Release date needs to be updated later on at the time of release.

File: README.md [hunk 2]
```diff
@@ -208,6 +208,15 @@ View additional information on configuring Windows images for bootstrap in the d
 Users can also include the ec2 server id in the node name by placing `%s` in the string passed to the `--chef-node-name` option. The %s is replaced by the ec2 server id dynamically. 
 e.g. `-N "www-server-%s" or  --chef-node-name "www-server-%s"`
 
+#### Tagging node in Chef
+
+Users can use option `--tag-node-in-chef` for tagging node in EC2 and chef both with `knife ec2 server create` command. If user does not pass option then it will be tagged only in EC2.
```
Comment (NimishaS) [inline README.md]: Small change: `If user does not pass this option, then the node will be tagged only in EC2.`

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,16 @@ Note: this log contains only changes from knife-ec2 release 0.8.0 and later
 prior to release 0.8.0, please visit the [source repository](https://github.com/chef/knife-ec2/commits).
 
 
+## Latest Release 0.16.0 (yyyy-mm-dd)
+* [knife-ec2:#503](https://github.com/chef/knife-ec2/pull/503) Update list of instance_types that support ebs-encryption.
+* [knife-ec2:#502](https://github.com/chef/knife-ec2/pull/502) Update bundler due to travis failure.
```
Comment (tas50) [inline CHANGELOG.md]: This can probably stay out of the changelog

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,16 @@ Note: this log contains only changes from knife-ec2 release 0.8.0 and later
 prior to release 0.8.0, please visit the [source repository](https://github.com/chef/knife-ec2/commits).
 
 
+## Latest Release 0.16.0 (yyyy-mm-dd)
```
Comment (tas50) [inline CHANGELOG.md]: Date?

Comment (piyushawasthi) [reply CHANGELOG.md]: Hi @tas50, as per comment of @NimishaS `Release date needs to be updated later on at the time of release.` https://github.com/chef/knife-ec2/pull/510#discussion-diff-146511732R8

File: CHANGELOG.md (hunk 3 above)
Comment (piyushawasthi) [reply CHANGELOG.md]: Remove `Update bundler due to travis failure.` from changelog. Thanks for review @tas50

**Pull request merged**